app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def self.update_likes_counter(post)
+    likes_count = Like.where(post:).count
+    post.update(LikesCounter: likes_count)
```

Comment:
- [ ] Kindly make sure to increment the counter of likes by using `post.increment!(:likes_counter)`